### PR TITLE
Csp 597 delete course directory and course delivery code and configuration

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs [hunk 1]
```diff
@@ -10,42 +10,25 @@ namespace SFA.DAS.Reservations.Application.Providers.Queries.GetProvider
...
-                        Ukprn = request.Ukprn
-                    });
-            }
+            var result = await _roatpApiClient.Get<GetRoatpProviderResponse>(
+                new GetProviderRequest
+                {
+                    Ukprn = request.Ukprn
+                });
 
+            provider = result;
```
Comment (cofaulco) [inline src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs:30]: this line could be removed

**Review by cofaulco — CHANGES_REQUESTED** [review]

**Review by cofaulco — APPROVED** [review]

File: src/EmployerFinance/SFA.DAS.EmployerFinance/InnerApi/Responses/GetProviderResponse.cs [hunk 2]
```diff
@@ -1,10 +1,10 @@
-﻿namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.ProviderCourses
+﻿namespace SFA.DAS.EmployerFinance.InnerApi.Responses
 {
     public class GetProviderResponse
     {
         public int Ukprn { get; set; }
-        public string LegalName { get; set; }
-        public string Website { get; set; }
+        public string Name { get; set; }
```
Comment (DevenKShah) [inline src/EmployerFinance/SFA.DAS.EmployerFinance/InnerApi/Responses/GetProviderResponse.cs:6]: Is this change required?

**Review by DevenKShah — APPROVED** [review]

File: src/EmployerFinance/SFA.DAS.EmployerFinance/InnerApi/Responses/GetProviderResponse.cs (hunk 2 above)
Comment (MarkFCain) [reply src/EmployerFinance/SFA.DAS.EmployerFinance/InnerApi/Responses/GetProviderResponse.cs:6]: it does appear to need that change.  For some reason, it was awaiting the fields 'LegalName' and 'Website' from roatp inner, but the fields are actually 'name' and 'ContactUrl', hence the change.    And Corey said this endpoint is  called from the EmployerFinance messagehandlers during the MA payment pull.   I presume such calls would get a null value for the unmapped fields at the moment.
